### PR TITLE
fix travis-ci build stage

### PR DESCRIPTION
**Pull request by luisan00**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
+- pip3 install -U sphinx-rtd-theme
+- pip3 install -U breathe
+- pip3 install -U exhale
 # Install ESP32 toochain following steps as described
 # in https://docs.espressif.com/projects/esp-idf/en/latest/get-started/#get-started-get-prerequisites
 - mkdir -p ~/esp
 - cd ~/esp
 - git clone --recursive https://github.com/espressif/esp-idf.git
 - cd ~/esp/esp-idf
+# - git checkout release/v4.0
```
Comment (jeandudey) [inline .travis.yml]:
```suggestion
- git clone --recursive https://github.com/espressif/esp-idf.git -b release/v4.1
- cd ~/esp/esp-idf
```

That way release 4.1 (latest stable one) can be used. Saves one line too :tada:

Comment (jeandudey) [reply .travis.yml]: Oops, my bad, it's v4.0

Comment (luisan00) [reply .travis.yml]: the latest release is v4.0

File: .travis.yml [hunk 2]
```diff
@@ -1,43 +1,48 @@
...
+      - libssl-dev
       - libncurses-dev
-      - flex
-      - bison
-
+      - doxygen
 
 before_install:
-- sudo apt-get -y install doxygen
+# - sudo apt-get -y install doxygen
```
Comment (jeandudey) [inline .travis.yml]:
```suggestion
```

It's now installed on `apt` section :wink:

**Comment by luisan00** [comment]
fix #116

**Pull request merged**